src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('tbody > tr');
+const headers = document.querySelectorAll('thead > tr > th');
+
+headers.forEach(item => {
+  item.addEventListener('click', sorted);
+});
+
+function sorted(ev) {
+  let sorter = null;
+
+  if (ev.target.innerText === 'Name') {
+    sorter = (a, b) => {
```

Comment:
Write one function Sorter, that will have switch case with all the possible options of column name.